### PR TITLE
Qoldev 77 hyperlinks on dfv

### PR DESCRIPTION
**Pull request by ghazal-salehi**
Applying button styles to Dfv portal
This includes default, active, hover states. Also the vertical spacing in the link list is fixed as well.

https://oss-uat.clients.squiz.net/community/getting-support-health-social-issue/support-victims-abuse/domestic-family-violence/i-want-to-help-someone

**Before:**
image

**After:**
image

**Before:**
image

**After:**
image

### CODE REVIEW

File: src/assets/_project/_blocks/franchise/DFV/qg-dfv.scss [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
+        text-decoration: underline !important;
+        text-decoration-thickness: 2px !important;
+    }
 }
 #qg-primary-content .dfv-page-introduction a, #qg-primary-content .dfv-content a:not(.qg-links-list__link) {
     color: $qg-blue-dark !important;
 }
+
+#qg-content #qg-primary-content .dfv-topics__link {
+    margin: 0 !important;
```
Comment (ThrawnCA) [inline src/assets/_project/_blocks/franchise/DFV/qg-dfv.scss:23]: Why does this need to be `!important`?

Comment (ghazal-salehi) [reply src/assets/_project/_blocks/franchise/DFV/qg-dfv.scss:23]: This is to override the default 2px margin. I can replace it with `@include qg-global-link-styles($margin: 0);`
I'll update it soon.

**Comment by ThrawnCA** [comment]
Can the build failures be looked into before approving and merging this? They might not be caused by this change, but I'd like to fix them before proceeding with confidence.

**Comment by ghazal-salehi** [comment]
> Can the build failures be looked into before approving and merging this? They might not be caused by this change, but I'd like to fix them before proceeding with confidence.

It's the footer feedback test that Steve fixed yesterday. It was merged this morning. I'll rebase my changes to make sure.

**Comment by ghazal-salehi** [comment]
Buiid was successful.

**Review by ThrawnCA — APPROVED** [review]

**Pull request merged**